src/domain/core/src/services/ingest/polling_ingest_service.rs

```diff
@@ -233,13 +219,6 @@ impl TemplateInvalidPatternError {
 // TODO: Revisit error granularity
 #[derive(Debug, Error)]
 pub enum PollingIngestError {
-    #[error(transparent)]
-    DatasetNotFound(
```

Comment:
Oh this is so cool that some errors are not even possible at this layer any more!